tasks.go

```diff
@@ -46,8 +46,8 @@ func (c *Client) PollTaskStatus(refURL string) (*ExecutionStatusResponse, error)
 	if refURL == "" {
 		return nil, errors.New("no taskRef provided to follow")
 	}
-	timer := time.NewTimer(c.retryIncrement)
-	t := time.NewTicker(1 * time.Second)
+	timer := time.NewTimer(4 * time.Minute)
```

Comment:
Do you really mean to poll for 4 minutes? We probably want to make sure this isn't blocking, because something else could time out along the way while waiting for this.

Reply:
This is part of the web request, so I think it should already be parallelized by Gorilla?

Reply:
Yea, I see what you're saying.